packages/suite/src/components/wallet/WalletLayout/AccountBanners/CloseableBanner.tsx

```diff
@@ -0,0 +1,46 @@
+import styled from 'styled-components';
+import { Button, Card, H3, variables } from '@trezor/components';
+import { Translation } from 'src/components/suite';
+import { ReactNode } from 'react';
+
+interface StyledCardProps {
+    color: string;
+}
+
+const StyledCard = styled(Card)<StyledCardProps>`
+    width: 100%;
+    margin-bottom: 48px;
+    padding: 24px 28px 24px 18px;
+    border-left: 10px solid ${({ color }) => color};
+    box-shadow: 0 2px 5px 0 ${({ theme }) => theme.BOX_SHADOW_BLACK_20};
```

Comment:
btw we try to have unified borders etc. Could this work?

```jsx
box-shadow: ${() => theme.boxShadowElevated};
```

Reply:
fixed in https://github.com/trezor/trezor-suite/pull/11099/commits/6ee47baa47af021ca155ca51626de835864a58f0